src/QuadraticProgram/QuadraticProgram.jl

```diff
@@ -356,10 +355,10 @@ function DiffOpt.forward_differentiate!(model::Model)
     partial_grads = if norm(Q) ≈ 0
         -IterativeSolvers.lsqr(LHS', RHS)
```

Comment:
This could also go through LinearSolve: https://github.com/SciML/LinearSolve.jl/blob/f2185983cfd4b25703c498975eb0174174bcc9e5/src/iterative_wrappers.jl#L55